CRM/Contribute/BAO/Contribution.php

```diff
@@ -4001,9 +4001,11 @@ public static function updateMembershipBasedOnCompletionOfContribution($contribu
         'is_test' => $membership['is_test'],
         'membership_type_id' => $membership['membership_type_id'],
         'membership_activity_status' => 'Completed',
+        'skipStatusCal' => 1,
```

Comment:
Note: API3 does: `if (!empty($params['id']) && !isset($params['skipStatusCal'])) { $params['skipStatusCal']=1 }`

We just add it in explicitly here to help with refactoring away from the API3 call to Membership::create in line 4081.